### PR TITLE
Make pre-commit happy

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

Fixes: https://github.com/thoth-station/thamos/issues/791

### CODE REVIEW

**Review by harshad16 — COMMENTED** [review]
/lgtm

**Comment by pacospace** [comment]
@fridex can we merge this?

File: Pipfile [hunk 1]
```diff
@@ -26,9 +26,14 @@ virtualenv = "*"
 
 [dev-packages]
 toml = "*"
+pytest = "*"
 pytest-timeout = "*"
 pytest-cov = "*"
-pytest = "*"
+hypothesis-auto = "*"
+hypothesis = "*"
```
Comment (fridex) [inline Pipfile:33]: hypothesis is not used in the sources.

File: Pipfile [hunk 2]
```diff
@@ -26,9 +26,14 @@ virtualenv = "*"
...
+pytest-mypy = "*"
+mypy = "*"
+pre-commit = "==2.0.1"
```
Comment (fridex) [inline Pipfile:36]: I would suggest leaving mypy and pre-commit on the testing environment - they are not direcly dependencies needed for the testsuite and we have container images in aicoe-ci that will test this.

**Review by fridex — COMMENTED** [review]
Some changes need broader context to have this properly in place. I will try to address these based on your changes.

**Comment by pacospace** [comment]
> Some changes need broader context to have this properly in place. I will try to address these based on your changes.

Can we close this?

**Pull request closed without merging**